packages/react/src/components/Slider/Slider.js

```diff
@@ -17,6 +19,31 @@ const defaultFormatLabel = (value, label) => {
   return typeof label === 'function' ? label(value) : `${value}${label}`;
 };
 
+/**
+ * Minimum time between processed "drag" events.
+ */
+const EVENT_THROTTLE = 16; // ms
+
+/**
+ * Event types that trigger "drags".
+ */
+const DRAG_EVENT_TYPES = new Set(['mousemove', 'touchmove']);
+
+/**
+ * Event types that trigger a "drag" to stop.
+ */
+const DRAG_STOP_EVENT_TYPES = new Set(['mouseup', 'touchend', 'touchcancel']);
+
+/**
+ * Constants used during keydown event processing.
+ */
+const ARROW_KEYS = Object.freeze({
```

Comment:
We have `matches` and `match` in our keyboard module to help out with keyboard events 👀 

https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/internal/keyboard/match.js#L15-L32

Reply:
Done!